src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ComboBoxTests.cs

```diff
@@ -2098,6 +2098,247 @@ public void ComboBox_DroppedDownProperty_Set_False_Invoke_RaiseAutomationPropert
+        {
+            using ComboBox comboBox = new()
+            {
+                Location = new(0, 0),
+            };
+            InitializeItems(comboBox, 10);
+
+            int numEvents = 0;
+            comboBox.SelectedIndexChanged += (x, y) => numEvents++;
+            comboBox.Sorted = false;
+            int numEventsExpected = 0;
+            numEvents = 0;
+            int selectedIndex = comboBox.SelectedIndex;
+            comboBox.SelectedIndex = index;
+            numEventsExpected++;
+
+            Assert.Equal(index, comboBox.SelectedIndex);
+            Assert.True(comboBox.Items[comboBox.SelectedIndex].Equals(comboBox.SelectedItem));
+            Assert.Equal(numEventsExpected, numEvents);
+        }
+
+        [WinFormsTheory]
+        [InlineData("item0", 0, 0, false)]
+        [InlineData("item1", 0, 1, false)]
+        [InlineData("item1", 1, 1, false)]
+        [InlineData("item1", 2, 1, false)]
+        [InlineData("item7", 2, 7, false)]
+        [InlineData("item7", 9, 7, false)]
+        [InlineData("item9", 0, 9, false)]
+        [InlineData("item9", 9, 9, false)]
+        [InlineData("Item9", 9, -1, false)]
+        [InlineData("item1984", 9, -1, false)]
+        [InlineData("item0", 0, 0, true)]
+        [InlineData("ITEM1", 0, 1, true)]
+        [InlineData("itEm1", 1, 1, true)]
+        [InlineData("item1", 2, 1, true)]
+        [InlineData("item7", 2, 7, true)]
+        [InlineData("item7", 9, 7, true)]
+        [InlineData("item9", 0, 9, true)]
+        [InlineData("item9", 9, 9, true)]
+        [InlineData("Item9", 9, 9, true)]
+        [InlineData("item1984", 9, -1, true)]
+        public void ComboBox_FindString(string value, int index, int expected, bool isExact)
+        {
+            using ComboBox comboBox = new()
+            {
+                Location = new(0, 0),
+            };
+            InitializeItems(comboBox, 10);
+
+            int actual = comboBox.FindStringExact(value, index, isExact);
+
+            Assert.Equal(expected, actual);
+        }
+
+        [WinFormsFact]
+        public void DerivedComboBox_Verify_OnMeasureItem_Receives_Correct_Arguments()
+        {
+            using DerivedComboBox derivedComboBox = new()
+            {
+                Location = new Point(0, 0),
+            };
+            derivedComboBox.CreateControl();
+            derivedComboBox.Items.AddRange(new object[]
+            {
+                "One",
+                "Two",
+                "Three"
+            });
+            derivedComboBox.Location = new Point(0, 50);
+            Assert.Equal(3, derivedComboBox.MeasureItemEventArgs.Count);
+
+            for (int i = 0; i < 3; i++)
+            {
+                MeasureItemEventArgs e = derivedComboBox.MeasureItemEventArgs[i];
+                Assert.NotNull(e.Graphics);
+
+                Assert.Equal(e.Index, i);
+
+                Assert.Equal(18, e.ItemHeight);
+
+                Assert.Equal(0, e.ItemWidth);
```

Comment:
Keep similar actions together
```suggestion
                Assert.NotNull(e.Graphics);
                Assert.Equal(e.Index, i);
                Assert.Equal(18, e.ItemHeight);
                Assert.Equal(0, e.ItemWidth);
```